irene_plugin_telegram_face/plugin_telegram_audio_input.py

```diff
@@ -25,7 +26,20 @@ class TelegramAudioInputPlugin(MagicPlugin):
     """
 
     name = 'telegram_input_audio'
-    version = '0.1.0'
+    version = '0.1.1'
+
+    config_comment = """
+    Настройки приёма голосовых сообщений из Telegram.
+
+    Доступны следующие параметры:
+    - `recogniseTextReply`  - слать сообщения с распознаным текстом из голосового сообщения
+    """
+    class _Config(TypedDict):
+        recogniseTextReply: bool
+        
+    config: _Config = {
+        'recogniseTextReply': True,        
```

Comment:
```suggestion
        'recogniseTextReply': False,
```

Не думаю, что это полезная функция для большинства пользователей, так что, скорее всего, не стоит её включать по-умолчанию.